src/components/FolderCard/FolderCard.jsx

```diff
@@ -51,7 +51,7 @@ export const FolderCard = ({
       default:
         return ""
     }
-  }, [pageType])
+  }, [pageType, siteName, category])
```

Comment:
As above, should be dependent on url as well?

Reply:
See above